### PR TITLE
Language Server Refactor

### PR DESCRIPTION
**Pull request by MaxStalker**
Closes: 
#206 
#204 

## Description
This shall fix delayed updates to deployed contracts. Now it will properly update diagnostics about missing/non-existing fields and contracts.

- [x] - Provide working solution that will fix the issue
- [x] - Check that code refreshes properly and does not affects language client
- [x] - Refactor code and remove all unnecessary console logs and `spaghetti` code 
- [x] - Update language server package to include latest Cadence changes

### CODE REVIEW

**Comment by MaxStalker** [comment]
@10thfloor can you take a look? :bow:

File: src/components/Arguments/index.tsx [hunk 1]
```diff
@@ -328,6 +329,7 @@ const Arguments: React.FC<ArgumentsProps> = (props) => {
           }
           resultType = ResultType.Contract;
           rawResult = await contractDeployment();
+          // await updateAccountDeployedCode();
```
Comment (10thfloor) [inline src/components/Arguments/index.tsx:332]: Dead code? Can we remove?

File: src/containers/Editor/components.tsx [hunk 2]
```diff
@@ -201,6 +224,12 @@ const EditorContainer: React.FC<EditorContainerProps> = ({
   const [code, setCode] = useState('');
   const [activeId, setActiveId] = useState(null);
 
+  const projectAccess = useProject()
+/*  console.log({
+    active: projectAccess.active,
+    account: projectAccess.project.accounts[active.index]
+  })*/
```
Comment (10thfloor) [inline src/containers/Editor/components.tsx:231]: More dead code.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

**Comment by 10thfloor** [comment]
@MaxStalker Looks good. I tested and it is working. Some minor cleanup and we can merge this.

**Comment by MaxStalker** [comment]
Screenshot from 2022-02-15 16-25-36

#204 issue is not the issue anymore, thanks to new language server package.

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**